common/types.go

```diff
@@ -266,3 +266,10 @@ const (
 	WebhookEventTypeDelete WebhookEventType = "delete"
 	WebhookEventTypeOther  WebhookEventType = "other"
 )
+
+type WebhookMessage interface {
```

Comment:
Can you add a comment to explain that this is the webhook that Hubspot sends for a subscription?